### PR TITLE
GC Featured link - Fix custom background color and adjust design pattern 

### PR DESCRIPTION
**Pull request by Ricokola**
Related to: wet189 and wet-185

### CODE REVIEW

File: components/gc-featured-link/gc-featured-link-en.html [hunk 1]
```diff
@@ -1,116 +1,96 @@
...
-title: "Featured link - Provisional feature"
+title: Featured link - Provisional feature
+description: Documentation and working example for the featured link - provisional feature
+language: en
+altLangPage: gc-featured-link-fr.html
+dateModified: 2021-07-02
+layout: no-container
 ---
-{% include alert-provisional.html siteroot="../../" %}
+<div class=:"container">
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-en.html:9]: ":" don't belong there

File: components/gc-featured-link/gc-featured-link-en.html [hunk 2]
```diff
@@ -1,116 +1,96 @@
...
+	<h3>Featured link (provisional v1.0)</h3>
+</div>
+
+<section class="provisional gc-featured-link">
 	<div class="container">
 		<div class="row">
 			<div class="col-xs-12">
-				<h2 class="wb-inv">Featured link</h2>
-				<p><a class="stretched-link" href="#">Featured link [optional]</a></p>
+				<h2 class="wb-inv">Featured</h2>
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-en.html:43]: This is visible based on the design pattern.

File: components/gc-featured-link/gc-featured-link-en.html [hunk 3]
```diff
@@ -1,116 +1,96 @@
...
+	<p>This adds a featured link to a page that your institution needs to feature prominently. It should be used to bring attention to warnings or notices.</p>
 
-<h3>Featured link (provisional v1.0)</h3>
-<div class="provisional gc-featured-link">
+	<h2>Working examples</h2>
+
+	<h3>Featured link (provisional v1.0)</h3>
+</div>
+
+<section class="provisional gc-featured-link">
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-en.html:39]: Add a note at the top or before this example explaining the heading sequence is not followed for the demo purpose. Be creative.

File: components/gc-featured-link/gc-featured-link-fr.html [hunk 4]
```diff
@@ -1,116 +1,96 @@
...
+	<h2>Exemple pratiques</h2>
+
+	<h3>Lien en vedette (Provisiore v1.0)</h3>
+</div>
+
+<section class="provisional gc-featured-link">
 	<div class="container">
 		<div class="row">
 			<div class="col-xs-12">
 				<h2 class="wb-inv">Lien en vedette</h2>
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-fr.html:43]: Need to be visible, see the design pattern

File: components/gc-featured-link/gc-featured-link-fr.html [hunk 5]
```diff
@@ -1,116 +1,96 @@
...
+	<p>Ceci ajoute un lien en vedette court et descriptif vers une page que votre institution doit mettre en évidence. Il doit être utilisé pour attirer l'attention sur les avertissements ou les avis.</p>
 
-<h3>Lien en vedette (Provisiore v1.0)</h3>
-<div class="provisional gc-featured-link">
+	<h2>Exemple pratiques</h2>
+
+	<h3>Lien en vedette (Provisiore v1.0)</h3>
+</div>
+
+<section class="provisional gc-featured-link">
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-fr.html:39]: See my comment for the English page, be creative and avoid confusion for the page structure through the heading sequence.

File: components/gc-featured-link/gc-featured-link-en.html [hunk 6]
```diff
@@ -1,116 +1,96 @@
...
-<div class="provisional gc-featured-link">
-	<div class="container">
-		<div class="row">
-			<div class="col-xs-12">
-				<p>Featured:&nbsp;</p>
-				<p><a class="stretched-link" href="#">[Promotion title]</a></p>
+<h3>Previous design patterns</h3>
+<details>
+	<summary>Previous pattern (Home page)</summary>
+	<p>Pattern using helpers:</p>
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-en.html:69]: 👍

File: components/gc-featured-link/gc-featured-link-fr.html [hunk 7]
```diff
@@ -1,116 +1,96 @@
...
-	<div class="container">
-		<div class="row">
-			<div class="col-xs-12">
-				<p>En vedette&nbsp;:&nbsp;</p>
-				<p><a class="stretched-link" href="#">[Titre de la promotion]</a></p>
+<h3>Modèles de conception précédents</h3>
+<details>
+	<summary>Modèle précédent (Page d'accueil)</summary>
+	<p>Modèle utilisant les "helpers":</p>
+	<div class="bg-gctheme opct-90">
```
Comment (duboisp) [inline components/gc-featured-link/gc-featured-link-fr.html:70]: 👍

**Review by duboisp — DISMISSED** [review]

File: components/gc-featured-link/gc-featured-link-en.html (hunk 2 above)
Comment (GormFrank) [reply components/gc-featured-link/gc-featured-link-en.html:44]: @duboisp The H2 has to be invisible as described in WET-185 and as per DTO's ask.

File: components/gc-featured-link/_base.scss [hunk 8]
```diff
@@ -1,17 +1,20 @@
 /* Provisional featured link component */
 
-.provisional.gc-featured-link {
+.gc-featured-link {
 	background-color: $bg-gctheme;
```
Comment (GormFrank) [inline components/gc-featured-link/_base.scss:4]: Everything seems perfect, except after looking at the beta template for ILP, I noticed that the font-family should be Lato, @Ricokola can you add this line please?
```suggestion
	background-color: $bg-gctheme;
	font-family: Lato, sans-serif;
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
One last little thing ans then it is ready

**Review by GormFrank — CHANGES_REQUESTED** [review]
Désolé @Ricokola, le `font-family` devrait aller juste au-dessus de `opacity` dans le `&.provisional`.

**Comment by Ricokola** [comment]
> 
> 
> Désolé @Ricokola, le `font-family` devrait aller juste au-dessus de `opacity` dans le `&.provisional`.

@GormFrank the change has been made and pushed to the PR

**Review by GormFrank — APPROVED** [review]

**Review by GormFrank — APPROVED** [review]
Pre-approved as long as code sections are fixed. Par exemple : 
```
<pre><code>&lt;div...
```

**Review by GormFrank — APPROVED** [review]

**Pull request merged**